### ISSUE TITLE
Would you be interested in running this as a hosted service for a monthly fee?

### ISSUE THREAD

**Issue by tonyxiao** [Brads3290/toggl_webhooks#9]

**Comment by Brads3290** [comment]
@tonyxiao I thought about it, but to be honest, not really. Aside from not being in a position to dedicate the time & effort to it, I think there are a few other problems:

- The entire service would be built around an undocumented private API that could be changed or removed at any time without warning. I think I'd feel a bit disingenuous if I took money for something that could disappear at any moment.
- For it to be worthwhile we'd have to make an effort to spread the word about it, like with any product I guess. The problem there is that Toggl probably won't like that. I can see them trying to make life difficult for us if we started doing it.
- If Toggl decides to play funny-buggers anyway, operating it as a service creates one big target for them to aim at. Take down the service by IP blocking, sending legal demands (even if unfounded, still requires resources to "fight back"), etc. and they stop everyone from using it. Much harder for them to do that if everyone is running it themselves.

I think for me it mostly comes down to the fact that it's building on top of such shaky ground, plus the time-and-effort issue. If you want to have a go at running it as a service, go for it!

**Comment by tonyxiao** [comment]
fair enough. If anything I hope it pushes Toggl toward finally building its own webhook then this would have served its purpose. But yea, definitely a time-effort investment still.
